Validator/LocaleValidator.php

```diff
@@ -79,9 +79,9 @@ public function validate($locale, Constraint $constraint)
         $locale = (string) $locale;
 
         if ($this->intlExtension) {
-            $primary = SymfonyLocale::getPrimaryLanguage($locale);
-            $region  = SymfonyLocale::getRegion($locale);
-            $locales = SymfonyLocale::getLocales();
+            $primary = \Locale::getPrimaryLanguage($locale);
+            $region  = \Locale::getRegion($locale);
```

Comment:
Felt weird too but that looks like the way to go. \Locale is part of ext-intl and Symfony Intl Component provides stubs if ext-intl is not loaded.